dotnet-client-libraries/BenchmarkDotNetWithUnitTests/Tests/StringConcatBenchmarkLiveTest.cs

```diff
@@ -0,0 +1,53 @@
+namespace Tests;
+
+public class StringConcatBenchmarkLiveTest : IClassFixture<BenchmarkFixture>
+{
+    private readonly BenchmarkFixture _benchmarkFixture;
+
+    public StringConcatBenchmarkLiveTest(BenchmarkFixture benchmarkFixture)
+    {
+        _benchmarkFixture = benchmarkFixture;
+    }
```

Comment:
We could simplify, shorten the code in the tests if we added in a property that returned the `Reports` array.
`private ImmutableArray<BenchmarkReport> BenchmarkReports => _benchmarkFixture.BenchmarkSummary.Reports;`

We could go a step further and just store that as a private readonly variable, but either way it would reduce the code footprint in each of the tests.